frontend/src/components/common/table/table.tsx

```diff
@@ -0,0 +1,58 @@
+import { Column, useTable } from 'react-table';
+import { FC } from 'common/types/types';
+import styles from './table/styles.module.scss';
+
+type Props = {
+  columns: Column[];
+  data: unknown[];
+};
+
```

Comment:
make it generic, we won't need use type casting every time when we are passing props